addons/binding/org.openhab.binding.plclogo/src/main/java/org/openhab/binding/plclogo/PLCLogoBindingConstants.java

```diff
@@ -0,0 +1,159 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.plclogo;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.TreeMap;
+
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link PLCLogoBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Alexander Falkenstern - Initial contribution
+ */
+public class PLCLogoBindingConstants {
+
+    public static final String BINDING_ID = "plclogo";
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID THING_TYPE_DEVICE = new ThingTypeUID(BINDING_ID, "device");
+    public static final ThingTypeUID THING_TYPE_ANALOG = new ThingTypeUID(BINDING_ID, "analog");
+    public static final ThingTypeUID THING_TYPE_DIGITAL = new ThingTypeUID(BINDING_ID, "digital");
+
+    // List of all Channel Type UIDs
+    public static final String ANALOG_CHANNEL_ID = "value";
+    public static final String DIGITAL_CHANNEL_ID = "state";
+    public static final String RTC_CHANNEL_ID = "rtc";
+
+    // List of all Channel configurations
+    public static final String INPUT_CHANNEL = "input";
+    public static final String OUTPUT_CHANNEL = "output";
+    public static final String ANALOG_DATE_CHANNEL = "date";
+    public static final String ANALOG_TIME_CHANNEL = "time";
+    public static final String ANALOG_NUMBER_CHANNEL = "number";
+
+    // LOGO! family definitions
+    public static final String LOGO_0BA7 = "0BA7";
+    public static final String LOGO_0BA8 = "0BA8";
+
+    // LOGO! diagnostics memory
+    public static final Integer LOGO_STATE = 984; // Diagnostics
+
+    @SuppressWarnings("serial")
+    private static final Map<?, String> LOGO_STATES_0BA7 = Collections.unmodifiableMap(new TreeMap<Integer, String>() {
+        {
+            // @formatter:off
```

Comment:
It was a very hard job to get messages produced by different Logo families. I can read only bitmasks from Logo PLC's for them. Means each bit represent one message. Sadly, i've no idea about mapping bit <-> message now. I was able to find meaning of 8 now. Maybe i or someone else get more clue about this next time. There will be good point to start. The next point, i have no idea now, how to use locales in java: It's my 4'th "program" :-)